### PR TITLE
METRON-1701: Update steps to run the pycapa on kerberized cluster.

### PR DESCRIPTION
**Pull request by MohanDV**
## Contributor Comments
Updated the steps to run Pycapa with Kerberos

## Pull Request Checklist

Thank you for submitting a contribution to Apache Metron.  
Please refer to our [Development Guidelines](https://cwiki.apache.org/confluence/pages/viewpage.action?pageId=61332235) for the complete guide to follow for contributions.  
Please refer also to our [Build Verification Guidelines](https://cwiki.apache.org/confluence/display/METRON/Verifying+Builds?show-miniview) for complete smoke testing guides.  

In order to streamline the review of the contribution we ask you follow these guidelines and ask you to double check the following:

### For all changes:
- [X] Is there a JIRA ticket associated with this PR? If not one needs to be created at [Metron Jira](https://issues.apache.org/jira/browse/METRON/?selectedTab=com.atlassian.jira.jira-projects-plugin:summary-panel).
- [X] Does your PR title start with METRON-XXXX where XXXX is the JIRA number you are trying to resolve? Pay particular attention to the hyphen "-" character.
- [X] Has your PR been rebased against the latest commit within the target branch (typically master)?

### For code changes:
- [ ] Have you included steps to reproduce the behavior or problem that is being changed or addressed?
- [ ] Have you included steps or a guide to how the change may be verified and tested manually?
- [ ] Have you ensured that the full suite of tests and checks have been executed in the root metron folder via:
  ```
  mvn -q clean integration-test install && dev-utilities/build-utils/verify_licenses.sh 
  ```

- [ ] Have you written or updated unit tests and or integration tests to verify your changes?
- [ ] If adding new dependencies to the code, are these dependencies licensed in a way that is compatible for inclusion under [ASF 2.0](http://www.apache.org/legal/resolved.html#category-a)?
- [ ] Have you verified the basic functionality of the build by building and running locally with Vagrant full-dev environment or the equivalent?

### For documentation related changes:
- [X] Have you ensured that format looks appropriate for the output in which it is rendered by building and verifying the site-book? If not then run the following commands and the verify changes via `site-book/target/site/index.html`:

  ```
  cd site-book
  mvn site
  ```

#### Note:
Please ensure that once the PR is submitted, you check travis-ci for build issues and submit an update to your PR as soon as possible.
It is also recommended that [travis-ci](https://travis-ci.org) is set up for your personal repository such that your branches are built there before submitting a pull request.

### CODE REVIEW

File: metron-sensors/pycapa/README.md [hunk 1]
```diff
@@ -239,25 +239,47 @@ The probe can be used in a Kerberized environment.  Follow these additional step
   * The keytab used is located at `/etc/security/keytabs/metron.headless.keytab`
   * The service principal is `metron@EXAMPLE.COM`
 
+    If it is not, ensure that you have `libsasl` or `libsasl2` installed.  On CentOS, this can be installed with the following command.
```
[the remark this answers is not in the document]
Comment (MohanDV) [reply metron-sensors/pycapa/README.md]: done

File: metron-sensors/pycapa/README.md [hunk 2]
```diff
@@ -239,25 +239,47 @@ The probe can be used in a Kerberized environment.  Follow these additional step
   * The keytab used is located at `/etc/security/keytabs/metron.headless.keytab`
   * The service principal is `metron@EXAMPLE.COM`
 
+    If it is not, ensure that you have `libsasl` or `libsasl2` installed.  On CentOS, this can be installed with the following command.   
+    ```
+    yum install -y cyrus-sasl cyrus-sasl-devel cyrus-sasl-gssapi
+    ```
+ 
+ 1. Use the non-binary wheel to install confluent-kafka-python.
```
[the remark this answers is not in the document]
Comment (MohanDV) [reply metron-sensors/pycapa/README.md]: Moved these instruction to the top as description .

File: metron-sensors/pycapa/README.md [hunk 3]
```diff
@@ -239,25 +239,47 @@ The probe can be used in a Kerberized environment.  Follow these additional step
...
+   
+    The Python client README (https://github.com/confluentinc/confluent-kafka-python) has an important note for Kerberos case:
+    ```
+    **NOTE:** The pre-built Linux wheels do NOT contain SASL Kerberos support.
+             If you need SASL Kerberos support you must install librdkafka and
+             its dependencies using the above repositories and then build
+             confluent-kafka from source.
+     ```
```
[the remark this answers is not in the document]
Comment (MohanDV) [reply metron-sensors/pycapa/README.md]: done

File: metron-sensors/pycapa/README.md [hunk 4]
```diff
@@ -54,8 +54,8 @@ General notes on the installation of Pycapa.
 
     ```
     export PREFIX=/usr
-    wget https://github.com/edenhill/librdkafka/archive/v0.9.4.tar.gz   -O - | tar -xz
-    cd librdkafka-0.9.4/
+    wget https://github.com/edenhill/librdkafka/archive/v0.11.5.tar.gz   -O - | tar -xz
+    cd librdkafka-0.11.5/
```
[the remark this answers is not in the document]
Comment (MohanDV) [reply metron-sensors/pycapa/README.md]: Yes it works for unsecured deployment as well.

**Comment by nickwallen** [comment]
+1 This looks great.  Thanks for making these updates.  Very useful!

**Pull request closed without merging**